multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -199,6 +236,55 @@ def _compute_soft_label_loss(
         )
         return loss
 
+    def _compute_intermediate_loss(
```

Comment:
`intermediate` usually implies more internal features inside the backbones. But here we only use the output features of backbones. Using this name is not accurate and can cause conflicts when we add losses for real intermediate features. I suggest using other names, e.g, `_compute_output_feature_loss`.

Reply:
@sxjscience Shall we change the name from *intermediate* to *output_feature*?

Reply:
Sounds good.

Reply:
Done